test/functional/ex_cmds/cd_spec.lua

```diff
@@ -0,0 +1,95 @@
+-- Specs for :cd, :tcd, :lcd
+
+local helpers = require('test.functional.helpers')
+local nvim, execute, eq, clear, eval, feed =
+  helpers.nvim, helpers.execute, helpers.eq, helpers.clear, helpers.eval, helpers.feed
+
+describe(':cd :tcd', function()
+  before_each(clear)
```

Comment:
Directory tree should be created in `before_each` and removed in `after_each`. Otherwise failing test leaves unneeded directories.